src/capi_frontend/capi.cpp

```diff
@@ -13,27 +13,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //*****************************************************************************
-#include "pocapi.h"  // NOLINT
-
 #include <cstdint>
 #include <memory>
 #include <string>
 
-#include "buffer.hpp"
-#include "inferenceparameter.hpp"
-#include "inferencerequest.hpp"
-#include "inferenceresponse.hpp"
-#include "inferencetensor.hpp"
-#include "model_service.hpp"
-#include "modelinstance.hpp"
-#include "modelinstanceunloadguard.hpp"
-#include "modelmanager.hpp"
-#include "profiler.hpp"
-#include "servablemanagermodule.hpp"
-#include "server.hpp"
-#include "server_options.hpp"
-#include "status.hpp"
-#include "timer.hpp"
+#include "../buffer.hpp"
```

Comment:
why this nested folder is added?

Reply:
It follows convention with tfs and kfs frontend into OVMS.